scenarioo-client/test/e2e/specs/manage_comparison.ts

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+import { scenario, step, useCase } from 'scenarioo-js';
+import * as Utils from '../util';
+import ComparisonDetailDialog from '../pages/comparisonDetailLogDialog';
+import ComparisonsPage from '../pages/comparisonsPage';
+
+useCase('Manage comparisons')
+    .description('View and manage comparisons')
+    .describe(() => {
+
+        beforeEach(async () => {
+            await Utils.startScenariooRevisited();
+        });
+
+        scenario('Filter comparisons')
+            .description('Comparisons can be filtered')
+            .it(async () => {
+                await ComparisonsPage.goToPage();
+                const comparisonsCount = ComparisonsPage.getNumberOfComparisons();
+                await step('comparisons page');
+
+                await ComparisonsPage.filterComparisons('To Projectstart');
+                await ComparisonsPage.assertNumberOfComparisons(5);
+                await step('filtered Comparisons for text \'To Projectstart\'');
```

Comment:
Use template strings so you don't have to escape

Reply:
I tried to use template string `filtered Comparisons for text 'To Projectstart'` but that led to an npm compile error. It worked with "filtered Comparisons for text 'To Projectstart'".